File: Assignments/MEAN/08_20 .11.2020/Assignment/Question_02/app/employee-details/employee.ts
export class Employee {
    empNo: number;
    empName: string;
    salary: number;
    designation: string;

    constructor(empNo: number, empName: string, salary: number, designation: string) {
        this.empNo = empNo;
        this.empName = empName;
        this.salary = salary;
        this.designation = designation;
    }
}

File: Assignments/Testing/01_25 .11.2020/src/com/app/Employee.java
package com.app;

public class Employee implements Payable, incrementable {

	double salary;
	int empId;
	String empName;
	double conSalary;

	@Override
	public double payment(double pay, double tax) {
		salary = pay - tax;
		return salary;
	}

	@Override
	public double paymentContract(double pay, double contractPay, double tax) {
		conSalary = pay + contractPay - tax;
		return conSalary;
	}
}
